examples/kubernetes/fleet-deployment.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: fleet-webserver
+  labels:
+    app: fleet-webserver
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: fleet-webserver
+  template:
+    metadata:
+      labels:
+        app: fleet-webserver
+    spec:
+      volumes:
+        - name: fleet-tls
+          secret:
+            secretName: fleet-tls
+      containers:
+      - name: fleet-webserver
+        image: kolide/fleet:latest
```

Comment:
`latest` is not great in k8s, because applying the deployment again won’t pull latest. k8s versioning of imagesis done through explicit version numbers. 

Make a note of that in the docs if you want to keep latest in the docs.

Reply:
Reference https://kubernetes.io/docs/concepts/configuration/overview/#container-images